[22.0.0] ZTS build failure

```
In file included from /home/rpmbuild/SPECS/remirepo/php/pecl/php-pecl-openswoole22/openswoole-22.0.0/ext-src/php_swoole_private.h:25,
                 from /home/rpmbuild/SPECS/remirepo/php/pecl/php-pecl-openswoole22/openswoole-22.0.0/ext-src/swoole_atomic.cc:17:
/home/rpmbuild/SPECS/remirepo/php/pecl/php-pecl-openswoole22/openswoole-22.0.0/ext-src/php_swoole_private.h: In function 'int php_swoole_check_reactor()':
./php_openswoole.h:58:22: error: 'openswoole_globals' was not declared in this scope; did you mean 'openswoole_globals_id'?
   58 | #define SWOOLE_G(v) (openswoole_globals.v)
      |                      ^~~~~~~~~~~~~~~~~~
/home/rpmbuild/SPECS/remirepo/php/pecl/php-pecl-openswoole22/openswoole-22.0.0/ext-src/php_swoole_private.h:1015:9: note: in expansion of macro 'SWOOLE_G'
 1015 |     if (SWOOLE_G(req_status) == PHP_SWOOLE_RSHUTDOWN_BEGIN) {
      |         ^~~~~~~~

```

```
make
/bin/bash /tmp/ext-openswoole/libtool --mode=compile g++ -I. -I/tmp/ext-openswoole -I/tmp/ext-openswoole/include -I/tmp/ext-openswoole/main -I/tmp/ext-openswoole -I/opt/php8/include/php -I/opt/php8/include/php/main -I/opt/php8/include/php/TSRM -I/opt/php8/include/php/Zend -I/opt/php8/include/php/ext -I/opt/php8/include/php/ext/date/lib -I/tmp/ext-openswoole -I/tmp/ext-openswoole/include -I/tmp/ext-openswoole/ext-src -I/tmp/ext-openswoole/thirdparty/hiredis  -DHAVE_CONFIG_H  -g -O2 -Wall -Wno-unused-function -Wno-deprecated -Wno-deprecated-declarations -std=c++11    -DENABLE_PHP_SWOOLE -DZEND_COMPILE_DL_EXT=1 -c /tmp/ext-openswoole/ext-src/php_swoole.cc -o ext-src/php_swoole.lo  -MMD -MF ext-src/php_swoole.dep -MT ext-src/php_swoole.lo
mkdir ext-src/.libs
 g++ -I. -I/tmp/ext-openswoole -I/tmp/ext-openswoole/include -I/tmp/ext-openswoole/main -I/tmp/ext-openswoole -I/opt/php8/include/php -I/opt/php8/include/php/main -I/opt/php8/include/php/TSRM -I/opt/php8/include/php/Zend -I/opt/php8/include/php/ext -I/opt/php8/include/php/ext/date/lib -I/tmp/ext-openswoole -I/tmp/ext-openswoole/include -I/tmp/ext-openswoole/ext-src -I/tmp/ext-openswoole/thirdparty/hiredis -DHAVE_CONFIG_H -g -O2 -Wall -Wno-unused-function -Wno-deprecated -Wno-deprecated-declarations -std=c++11 -DENABLE_PHP_SWOOLE -DZEND_COMPILE_DL_EXT=1 -c /tmp/ext-openswoole/ext-src/php_swoole.cc -MMD -MF ext-src/php_swoole.dep -MT ext-src/php_swoole.lo  -fPIC -DPIC -o ext-src/.libs/php_swoole.o
In file included from /tmp/ext-openswoole/ext-src/php_swoole_private.h:25,
                 from /tmp/ext-openswoole/ext-src/php_swoole_cxx.h:19,
                 from /tmp/ext-openswoole/ext-src/php_swoole.cc:16:
/tmp/ext-openswoole/ext-src/php_swoole_private.h: In function ‘int php_swoole_check_reactor()’:
./php_openswoole.h:58:22: error: ‘openswoole_globals’ was not declared in this scope; did you mean ‘openswoole_globals_id’?
   58 | #define SWOOLE_G(v) (openswoole_globals.v)
      |                      ^~~~~~~~~~~~~~~~~~
```

Same...

--- a/php_openswoole.h
+++ b/php_openswoole.h
@@ -55,6 +55,10 @@ ZEND_END_MODULE_GLOBALS(openswoole)
 
 extern ZEND_DECLARE_MODULE_GLOBALS(openswoole);
 
+#ifdef ZTS
+#define SWOOLE_G(v) TSRMG(openswoole_globals_id, zend_openswoole_globals *, v)
+#else
 #define SWOOLE_G(v) (openswoole_globals.v)
+#endif
 
 #endif /* PHP_SWOOLE_H */

